upgrade-notes/backend_20240731_200701.md

```diff
@@ -0,0 +1,23 @@
+#### [shopsys] role `ROLE_ALL_API` sees all customer user orders from common customer ([#3296](https://github.com/shopsys/shopsys/pull/3296))
```

Comment:
please without `[shopsys]` and preferably in a form like "do this" - e.g., "allow ROLE_API_ALL to see all customer user orders from common customer", or something like that

Reply:
Thx. Fixed: https://github.com/shopsys/shopsys/pull/3296/commits/98de48f94c15fa838c9ed80ff49ab5ee47330cbd